client/src/format.js

```diff
@@ -86,21 +83,17 @@ const formatAccount = accountInfo => {
 
 // FORMAT MOSAICS
 const formatMosaics = mosaics => {
-  mosaics.map(mosaic => {
+  return mosaics.map(mosaic => {
     mosaic.hex = mosaic.id.toHex()
     mosaic.amount = mosaic.amount.compact()
```

Comment:
I think it's missing `return mosaic` inside the `map()`

Reply:
Good point, thanks!